integration/kubeseal_test.go

```diff
@@ -209,3 +209,26 @@ var _ = Describe("kubeseal --fetch-cert", func() {
 			Should(Equal(certs))
 	})
 })
+
+var _ = Describe("kubeseal --version", func() {
+	var c corev1.CoreV1Interface
```

Comment:
I think `c` is unused here?

Reply:
Ah, hadn't realised it's not running fmt/vet over this.